scipy/linalg/_decomp.py

```diff
@@ -123,6 +124,11 @@ def eig(a, b=None, left=False, right=True, overwrite_a=False,
 
     where ``.H`` is the Hermitian conjugation.
 
+    The documentation is written assuming array arguments of are of specified
```

Comment:
```suggestion
    The documentation is written assuming array arguments are of specified
```
Otherwise, how does this look to you?